MozillaFoundation/foundation.mozilla.org
# Add middleware to redirect pt-br to correct locale path

# Description

This PR will add a middleware function to essentially make the pt-BR locale case-insensitive. If the request's locale path is `pt-br` it will redirect users to the correct path `pt-BR` instead of throwing error. It's the most frequent error on our site, and is worthy of accommodation.

To test:
1) Check out branch
2) Navigate to any path with all lower case path '/pt-br/'
3) Confirm you are redirected to `/pt-BR/` locale path instead of getting an error

Related PRs/issues: TP1-1479

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-1558)
